source/overview/faq.rst

```diff
@@ -31,6 +31,13 @@ We have done performance testing of 60,000 concurrent users and 60 million posts
 
 Mattermost provides an open source, well-documented `load test simulator <https://github.com/mattermost/mattermost-load-test>`_ to verify that your Mattermost deployment can achieve the stated scale benchmarks ahead of production deployment.
 
+How do I deploy the open source Mattermost Team Edition under an MIT license?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The open source Mattermost Team Edition is functionally identical to the commercial Mattermost Enterprise Edition in its free “team mode”, but there is no ability to unlock enterprise features. It deploys as single Linux binary with MySQL or PostgreSQL under an MIT license.
```

Comment:
Can we add a sentence about what we recommend, similar to how GitLab does?

E.g. "We generally recommend installing Enterprise Edition, even if you're not sure if you will ever install a license. However, if you only want to install software with a fully open source code base, then Team Edition is the best choice for you." 

Not sure if "fully open source code base" sounds exactly right, but I think we should have something along the lines of the example so we're giving a clear recommendation on when to use each.